extensions/data-plane/data-plane-api/src/main/java/org/eclipse/dataspaceconnector/dataplane/api/response/ResponseFunctions.java

```diff
@@ -28,25 +28,28 @@
  */
 public final class ResponseFunctions {
 
+    private static final String DATA = "data";
```

Comment:
I haven checked it, but in the previous (main) logic, data gets encoded twice (provider side and then again in the consumer side), so we did need to run "double decode" on the data response object. 

I could not find an evidence that this behaviour was improved ...

Reply:
good catch indeed. 
I removed this data stuff so that only the raw data is now returned in output.

Reply:
top notch.